docs/sections/user_guide/uw_yaml/field_table_yaml.rst

```diff
@@ -0,0 +1,52 @@
+.. _defining_a_field_table:
+
+Defining a ``field_table``
+==========================
+
+The :ufs-weather-model:`UFS Weather Model<>` requires as one of its input files (:weather-model-io:`documented here<model-configuration-files>`) a ``field_table`` file. While the format required by the ufs-weather-model is not a YAML file, with ``uwtools`` package, the ``field_table`` entries may be managed as a YAML file, and written to the expected structure (:weather-model-io:`documented here<field-table-file>`).
+
+To generate a given ``field_table`` entry with the form:
+
+.. code-block:: text
+
+   "TRACER", "atmos_mod", "sphum"
+             "longname",     "specific humidity"
+             "units",        "kg/kg"
+             "profile_type", "fixed", "surface_value=3.e-6" /
+
+Entries can be manage in YAML with the form:
+
+.. code-block:: text
+
+   sphum:
+     longname: "specific humidity"
```

Comment:
Doesn't really matter, but FYI the quotes here are optional. YAML will parse it as the same string either way. The quotes are kind of voodoo in this case. I only bring it up because it may obscure an understanding of how YAML really works, and the power of using quotes when they're _really_ needed, i.e. to force a value that normally would be parsed as something else to be parsed as a string.